MBBSEmu/HostProcess/Structs/ModuleStruct.cs

```diff
@@ -20,6 +20,7 @@ public class ModuleStruct
         public FarPtr finrou { get; set; }
 
         public const ushort Size = 61;
+        public const ushort DESCRP_SIZE = 25;
```

Comment:
Have this be the assigned length as well in the `byte[] Data` getters and setters for `descrp` for consistency